src/commands/channels/deactivate.ts

```diff
@@ -0,0 +1,108 @@
+/*
+  * Copyright 2022, Nuance, Inc. and its contributors.
+  * All rights reserved.
+  *
+  * This source code is licensed under the Apache-2.0 license found in
+  * the LICENSE file in the root directory of this source tree.
+  */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as MixFlags from '../../utils/flags'
+import * as ChannelsAPI from '../../mix/api/channels'
+import {DomainOption} from '../../utils/validations'
+import {MixClient, MixResponse} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+import {ChannelsDeactivateParams} from '../../mix/api/channels-types'
+import {cli} from 'cli-ux'
```

Comment:
Move the "cli" import to the first section with the other third-party imports. Sort.